pyonfx/convert.py

```diff
@@ -109,7 +126,7 @@ def alpha_ass_to_dec(alpha_ass: str) -> int:
             ) from e
 
     @staticmethod
-    def alpha_dec_to_ass(alpha_dec: Union[int, float]) -> str:
+    def alpha_dec_to_ass(alpha_dec: float) -> str:
```

Comment:
The accepted type here is also integer...
Isn't it more correct to also write `int`?

Reply:
Actually, I thought we should only accepts float in these conversions but `int` only here makes more sense. Since it's a alpha ASS value, you can't use a float anyway. What do you think?

Reply:
It does make sense to pass a float if `alpha_dec_to_ass` is used inside a Frame-per-Frame effect using FrameUtility.
In those cases, interpolation would return a float, so it is convenient to also pass floats.